extensions/amp-mega-menu/OWNERS.yaml

```diff
@@ -0,0 +1,5 @@
+# For an explanation of the OWNERS.yaml rules and syntax, see:
+# https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
+
+- leafsy
+- ampproject/wg-ui-and-a11y
```

Comment:
Update: The OWNERS bot has been migrate to [this JSON format]. Please replace this file with `extensions/amp-mega-menu/OWNERS` containing:

```javascript
// For an explanation of the OWNERS rules and syntax, see:
// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example

{
  rules: [
    { name: 'leafsy' },
    { name: 'ampproject/wg-ui-and-a11y' }
  ]
}
```